File: examples/museum/oseg_generated/typescript-node/GetMuseumHoursDefaultExample.ts
import * as fs from 'fs';
import * as apis from "openapimuseum_client/api/apis"
import * as models from "openapimuseum_client/model/models"

const apiCaller = new apis.OperationsApi();

const startDate = "2023-02-23";
const page = 2;
const limit = 15;

apiCaller.getMuseumHours(
    startDate,
    page,
    limit,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Operations#getMuseumHours:");
  console.log(error.body);
});

File: examples/museum/oseg_generated/ruby/GetMuseumHoursDefaultExample.rb
require "openapimuseum_client"

OpenApiMuseumClient.configure do |config|
end

begin
    response = OpenApiMuseumClient::OperationsApi.new.get_museum_hours(
        {
            start_date: Date.parse("2023-02-23").to_date,
            page: 2,
            limit: 15,
        },
    )

    p response
rescue OpenApiMuseumClient::ApiError => e
    puts "Exception when calling Operations#get_museum_hours: #{e}"
end
